src/lib/pages/contract-details/index.tsx

```diff
@@ -304,14 +305,18 @@ const ContractDetails = observer(() => {
   useWasmConfig({ shouldRedirect: true });
   const router = useRouter();
   const { validateContractAddress } = useValidateAddress();
+
   const contractAddressParam = getFirstQueryParam(
     router.query.contractAddress
   ) as ContractAddr;
+  // TODO: fix assertion later
+  const tab = getFirstQueryParam(router.query.tab) as TabIndex;
+
   const contractData = useContractData(contractAddressParam);
 
   useEffect(() => {
-    if (router.isReady) track(AmpEvent.TO_CONTRACT_DETAIL);
-  }, [router.isReady]);
+    if (router.isReady && tab) track(AmpEvent.TO_CONTRACT_DETAIL, { tab });
```

Comment:
Do you intend to prevent triggering when `tab` value is null or undefined?

Reply:
Yes